CMakeLists.txt

```diff
@@ -283,11 +283,6 @@ if(LIB_INSTALL_DIR)
     set(CMAKE_INSTALL_LIBDIR "${LIB_INSTALL_DIR}")
 endif()
 
-if (NOT EXISTS "${CMAKE_CURRENT_SOURCE_DIR}/framework/CMakeLists.txt")
-    message(FATAL_ERROR "${CMAKE_CURRENT_SOURCE_DIR}/framework/CMakeLists.txt not found. Run `git submodule update --init` from the source tree to fetch the submodule contents.")
-endif()
-add_subdirectory(framework)
-
```

Comment:
To make our future life during updates easier you could add to the commit message that this commit reverts some of the work of the commits:
469f7811fac98617e544aac2b1b2db56040c95db 
and 
f9bbe0de4c97ca532d052106bb471756e55742ae

Or even just add revert commits of these to commits